src/sound/soundbase.h

```diff
@@ -49,6 +49,7 @@ enum EMidiCtlType
     Solo,
     Mute,
     MuteMyself,
+    OurFader, // Proposed addition: a MidiCtrlType for our own fader level
```

Comment:
```suggestion
    MyFader, // Proposed addition: a MidiCtrlType for our own fader level
```

I think that's more in line with the code otherwise.